packages/teams-js/src/public/pages.ts

```diff
@@ -14,7 +14,14 @@ import { isNullOrUndefined } from '../internal/typeCheckUtilities';
 import { createTeamsAppLink } from '../internal/utils';
 import { app } from './app';
 import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
-import { FrameInfo, ShareDeepLinkParameters, TabInformation, TabInstance, TabInstanceParameters } from './interfaces';
+import {
```

Comment:
Your new capability should also have e2e tests written for it. The pages e2e tests are in `pages.json`

Reply:
Yes, will do, and this is because those compat tests should add after the web hub-sdk messageHandler or those tests will fail.